.storybook/preview.tsx

```diff
@@ -81,6 +81,16 @@ const getTheme = (
     };
 };
 
+const findAccessibilityAddonButton = (): HTMLButtonElement | undefined => {
```

Comment:
Search for the button to run the a11y tests in the storybook UI and click it.

![test](https://github.com/Telefonica/mistica-web/assets/25785151/faca0ef1-a4aa-47c1-ad85-0909074882f5)